src/main/java/com/htv3/htv3onlinemusic/controller/userprofile/UserController.java

```diff
@@ -0,0 +1,36 @@
+package com.htv3.htv3onlinemusic.controller.userprofile;
+
+import com.htv3.htv3onlinemusic.model.User;
+import com.htv3.htv3onlinemusic.service.user.IUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    private IUserService userService;
+
```

Comment:
Có thể tạo thêm endpoint lấy ra toàn bộ users trong hệ thống nhưng có secrity constraint là role Admin